platform/packaging/build/faiss/Dockerfile

```diff
@@ -95,20 +111,22 @@ RUN patch -p1 --input=/workspace/build/faiss-1.7.4-setuppy-include-binaries.patc
 RUN python3 -m venv ${VENV_PATH}
 RUN ${PIP_VENV} install numpy
 RUN ${PIP_VENV} install swig
+RUN ${PIP_VENV} install build
 RUN ${PIP_VENV} install wheel
 
+
 ###
 #
 # This is needed to use the Ninja generator
 # Unfortunately, I was not able to get the later make operations to work with ninja
 #
-#      -D CMAKE_ARGS:STRING="-d stats" \
+#      -D CMAKE_ENVS:STRING="-d stats" \
 #      -G Ninja \
 #      -DCMAKE_INSTALL_LIBDIR=lib \
 
 RUN cmake --fresh \
-      -S /workspace/faiss-1.7.4 \
-      -B _build \
+      -S /workspace/source/faiss-${FAISS_VERSION} \
```

Comment:
Path should have "sources" instead of "source".

Reply:
agree thanks.